### DISCUSSION TITLE
GCP Marketplace deployment won't allow second interface for sensor

### DISCUSSION THREAD

**Discussion by pwdutoit** [Security-Onion-Solutions/securityonion#11273] [2.4] [answered]
### Version

2.4.10

### Installation Method

Other (please provide detail below)

### Description

installation

### Installation Type

Distributed

### Location

cloud

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

32GB

### Storage for /

256GB

### Storage for /nsm

15TB

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

When trying to deploy a sensor instance on GCP, the deployment template does not allow the addition of a secondary nic on the monitoring VPC. 

Documentation says:
- (Distributed “Sensor” node or Single-Node grid only) Add a second Network interface and select the monitoring VPC network, and the appropriate subnet. No external ephemeral IP is necessary for this interface. Specify the network tag so-collector for this VM.

but it looks like the Terraform template defined 1/1 as MAX interfaces. After deployment I also couldn't manually add a NIC to the VM instance created :(

Thanks for this great project and your time!

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by jertel** [comment]
Thanks for raising this issue. We are aware of the inability to add the second NIC during deployment (there's a note about this on the post-deployment / next steps page). However, the inability to add a second NIC after deployment is unexpected. We'll investigate.

**Reply by jertel** [reply, marked as answer]
This has been resolved:

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/16639611/79d60025-86db-4665-ad20-32e15397af75)

**Reply by pwdutoit** [reply]
Perfect! Thank you so much for this.
